src/include/daos/common.h

```diff
@@ -80,6 +80,9 @@ struct daos_sgl_idx {
 	daos_off_t	iov_offset; /** byte offset of iov buf */
 };
 
+#define DF_SGL_IDX "{idx: %d, offset: "DF_U64"}"
+#define DP_SGL_IDX(i) (i)->iov_idx, (i)->iov_offset
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
I'm not sure what to do about these issues.